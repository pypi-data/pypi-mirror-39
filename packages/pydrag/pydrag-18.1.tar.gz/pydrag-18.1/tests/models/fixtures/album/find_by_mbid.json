{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&autocorrect=1&format=json&lang=en&mbid=6defd963-fe91-4550-b18e-82c685603c2b&method=album.getInfo"
            },
            "response": {
                "body": {
                    "string": "{\"album\":{\"name\":\"A Night at the Opera\",\"artist\":\"Queen\",\"mbid\":\"CENSORED95b2\",\"url\":\"https://www.last.fm/music/Queen/A+Night+at+the+Opera\",\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"mega\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"\"}],\"listeners\":\"605690\",\"playcount\":\"8620142\",\"tracks\":{\"track\":[{\"name\":\"Death on Two Legs (Dedicated to ...)\",\"url\":\"https://www.last.fm/music/Queen/_/Death+on+Two+Legs+(Dedicated+to+...)\",\"duration\":\"223\",\"@attr\":{\"rank\":\"1\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"Lazing on a Sunday Afternoon\",\"url\":\"https://www.last.fm/music/Queen/_/Lazing+on+a+Sunday+Afternoon\",\"duration\":\"67\",\"@attr\":{\"rank\":\"2\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"I'm in Love With My Car\",\"url\":\"https://www.last.fm/music/Queen/_/I%27m+in+Love+With+My+Car\",\"duration\":\"185\",\"@attr\":{\"rank\":\"3\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"You're My Best Friend\",\"url\":\"https://www.last.fm/music/Queen/_/You%27re+My+Best+Friend\",\"duration\":\"170\",\"@attr\":{\"rank\":\"4\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"'39\",\"url\":\"https://www.last.fm/music/Queen/_/%2739\",\"duration\":\"210\",\"@attr\":{\"rank\":\"5\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"Sweet Lady\",\"url\":\"https://www.last.fm/music/Queen/_/Sweet+Lady\",\"duration\":\"241\",\"@attr\":{\"rank\":\"6\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"Good Company\",\"url\":\"https://www.last.fm/music/Queen/_/Good+Company\",\"duration\":\"188\",\"@attr\":{\"rank\":\"7\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"Seaside Rendezvous\",\"url\":\"https://www.last.fm/music/Queen/_/Seaside+Rendezvous\",\"duration\":\"134\",\"@attr\":{\"rank\":\"8\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"The Prophet's Song\",\"url\":\"https://www.last.fm/music/Queen/_/The+Prophet%27s+Song\",\"duration\":\"497\",\"@attr\":{\"rank\":\"9\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"Love of My Life\",\"url\":\"https://www.last.fm/music/Queen/_/Love+of+My+Life\",\"duration\":\"218\",\"@attr\":{\"rank\":\"10\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"Bohemian Rhapsody\",\"url\":\"https://www.last.fm/music/Queen/_/Bohemian+Rhapsody\",\"duration\":\"356\",\"@attr\":{\"rank\":\"11\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}},{\"name\":\"God Save the Queen\",\"url\":\"https://www.last.fm/music/Queen/_/God+Save+the+Queen\",\"duration\":\"64\",\"@attr\":{\"rank\":\"12\"},\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Queen\",\"mbid\":\"CENSOREDf1a7\",\"url\":\"https://www.last.fm/music/Queen\"}}]},\"tags\":{\"tag\":[{\"name\":\"classic rock\",\"url\":\"https://www.last.fm/tag/classic+rock\"},{\"name\":\"albums I own\",\"url\":\"https://www.last.fm/tag/albums+I+own\"},{\"name\":\"rock\",\"url\":\"https://www.last.fm/tag/rock\"},{\"name\":\"70s\",\"url\":\"https://www.last.fm/tag/70s\"},{\"name\":\"hard rock\",\"url\":\"https://www.last.fm/tag/hard+rock\"}]},\"wiki\":{\"published\":\"29 Nov 2008, 18:16\",\"summary\":\"A Night at the Opera is a 1975 album by English rock band Queen. It was produced by Roy Thomas Baker and Queen, and reportedly was, at the time of its release, the most expensive album ever made. It was originally released by EMI in the UK where it topped the charts for nine weeks, a record at the time, and Elektra Records in the United States where the album peaked at #4 and has been certified Triple Platinum (three million copies sold).\\nThe album takes its name from the Marx Brothers film of the same name <a href=\\\"http://www.last.fm/music/Queen/A+Night+at+the+Opera\\\">Read more on Last.fm</a>.\",\"content\":\"A Night at the Opera is a 1975 album by English rock band Queen. It was produced by Roy Thomas Baker and Queen, and reportedly was, at the time of its release, the most expensive album ever made. It was originally released by EMI in the UK where it topped the charts for nine weeks, a record at the time, and Elektra Records in the United States where the album peaked at #4 and has been certified Triple Platinum (three million copies sold).\\nThe album takes its name from the Marx Brothers film of the same name, which the band watched one night at the studio complex when recording. The 1976 follow-up album, A Day at the Races, was also named after a Marx brothers film.\\n\\nAt the time the most expensive album ever recorded, Brian May has asserted in subsequent years that the band would have disbanded had it not been successful. Upon release, A Night at the Opera was a commercial and critical success, debuting at #1 in the UK charts and remaining there for nine weeks. In the US, it debuted at #4, the band's strongest showing at that time. Reviews were generally positive. Rolling Stone wrote: \\\"What sets them apart is their selection of unlikely effects: acoustic piano, harp, acapella vocals, no synthesizers. Coupled with good songs. Queen's obviously the strongest contender in its field.\\\" Melody Maker called the album a \\\"must-have,\\\" encouraging listeners to \\\"turn it up loud and enjoy,\\\" while the Winnipeg Free Press wrote: \\\"The group's potential is practically limitless, indicating that Queen is destined to finally take its place among the small handful of truly major acts working in rock today.\\\" Robert Christgau, however, was less enthusiastic, giving the album a \\\"B-\\\" rating. Later reviews in notable publications such as Allmusic, Mojo and Q Magazine, have seen the album receive five-star ratings. Allmusic wrote: \\\"It's prog rock with a sense of humor as well as dynamics, and Queen never bettered their approach anywhere else.\\\" Mojo called the album \\\"An imperial extravaganza, a cornucopia; a band of hungrily competitive individualists on a big roll of friendship and delight.\\\" Q were similarly receptive, writing: \\\"It's a record to which all four contributed fine songs, and one of extremes -- among the crashing rock and proggy wigouts were love songs, acoustic whimsy, a trad-jazz pastiche and a brace of vaudeville show tunes.\\\" Uncut awarded the album only 3/5 stars, but offered a mostly positive review, commending \\\"the extent of the band's barmy diversity.\\\"\\n\\nSince its release, A Night at the Opera has frequently been cited as one of Queen's finest works. In particular, Bohemian Rhapsody, the album's centerpiece, received acclaim and is still considered among the greatest songs ever recorded. The British public voted it the 13th greatest album of all time in a 2004 Channel 4 poll, and the 9th greatest number one album of all time in a 2006 BBC poll. The album has also ranked highly in international polls and has frequently appeared in \\\"greatest albums\\\" lists reflecting the opinions of critics.\\n\\nA Night at the Opera appears alongside other Queen albums Queen II and Sheer Heart Attack in the book 1001 Albums You Must Hear Before You Die.\\n\\nThe album was first time re-released in the US on Hollywood Records on 3 September 1991 with two bonus remixes, as part of a complete re-release of all Queen albums.\\n\\nOn 30 April 2002 the album was again re-released on DVD-Audio with a 5.1-channel mix in Dolby Digital and DTS surround sound.\\n\\nOn 21 November 2005 it was once more re-released to celebrate the 30th anniversary of the album and its first single, \\\"Bohemian Rhapsody\\\". This release is accompanied by a DVD-Video disc with the same track listing featuring the original videos, old and new concert footage (including \\\"'39\\\" from the Queen + Paul Rodgers tour and Brian May on the roof of Buckingham Palace playing \\\"God Save the Queen\\\") and audio commentary by all four bandmembers. It was on this commentary (and on In the Studio with Redbeard, which devoted an episode to A Night at the Opera) that May stated that had the album not been a success, Queen certainly would have disbanded. <a href=\\\"http://www.last.fm/music/Queen/A+Night+at+the+Opera\\\">Read more on Last.fm</a>. User-contributed text is available under the Creative Commons By-SA License; additional terms may apply.\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "9667"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:27:38 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
