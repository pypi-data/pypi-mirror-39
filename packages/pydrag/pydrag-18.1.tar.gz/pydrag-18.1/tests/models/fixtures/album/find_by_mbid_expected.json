{
    "artist": {
        "name": "Queen"
    },
    "image": [
        {
            "size": "small",
            "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
        },
        {
            "size": "medium",
            "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
        },
        {
            "size": "large",
            "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
        },
        {
            "size": "extralarge",
            "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
        },
        {
            "size": "mega",
            "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
        },
        {
            "size": "",
            "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
        }
    ],
    "listeners": 605690,
    "mbid": "CENSORED95b2",
    "name": "A Night at the Opera",
    "playcount": 8620142,
    "tags": [
        {
            "name": "classic rock",
            "url": "https://www.last.fm/tag/classic+rock"
        },
        {
            "name": "albums I own",
            "url": "https://www.last.fm/tag/albums+I+own"
        },
        {
            "name": "rock",
            "url": "https://www.last.fm/tag/rock"
        },
        {
            "name": "70s",
            "url": "https://www.last.fm/tag/70s"
        },
        {
            "name": "hard rock",
            "url": "https://www.last.fm/tag/hard+rock"
        }
    ],
    "tracks": [
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 223,
            "name": "Death on Two Legs (Dedicated to ...)",
            "rank": 1,
            "url": "https://www.last.fm/music/Queen/_/Death+on+Two+Legs+(Dedicated+to+...)"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 67,
            "name": "Lazing on a Sunday Afternoon",
            "rank": 2,
            "url": "https://www.last.fm/music/Queen/_/Lazing+on+a+Sunday+Afternoon"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 185,
            "name": "I'm in Love With My Car",
            "rank": 3,
            "url": "https://www.last.fm/music/Queen/_/I%27m+in+Love+With+My+Car"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 170,
            "name": "You're My Best Friend",
            "rank": 4,
            "url": "https://www.last.fm/music/Queen/_/You%27re+My+Best+Friend"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 210,
            "name": "'39",
            "rank": 5,
            "url": "https://www.last.fm/music/Queen/_/%2739"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 241,
            "name": "Sweet Lady",
            "rank": 6,
            "url": "https://www.last.fm/music/Queen/_/Sweet+Lady"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 188,
            "name": "Good Company",
            "rank": 7,
            "url": "https://www.last.fm/music/Queen/_/Good+Company"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 134,
            "name": "Seaside Rendezvous",
            "rank": 8,
            "url": "https://www.last.fm/music/Queen/_/Seaside+Rendezvous"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 497,
            "name": "The Prophet's Song",
            "rank": 9,
            "url": "https://www.last.fm/music/Queen/_/The+Prophet%27s+Song"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 218,
            "name": "Love of My Life",
            "rank": 10,
            "url": "https://www.last.fm/music/Queen/_/Love+of+My+Life"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 356,
            "name": "Bohemian Rhapsody",
            "rank": 11,
            "url": "https://www.last.fm/music/Queen/_/Bohemian+Rhapsody"
        },
        {
            "artist": {
                "mbid": "CENSOREDf1a7",
                "name": "Queen",
                "url": "https://www.last.fm/music/Queen"
            },
            "duration": 64,
            "name": "God Save the Queen",
            "rank": 12,
            "url": "https://www.last.fm/music/Queen/_/God+Save+the+Queen"
        }
    ],
    "url": "https://www.last.fm/music/Queen/A+Night+at+the+Opera",
    "wiki": {
        "content": "A Night at the Opera is a 1975 album by English rock band Queen. It was produced by Roy Thomas Baker and Queen, and reportedly was, at the time of its release, the most expensive album ever made. It was originally released by EMI in the UK where it topped the charts for nine weeks, a record at the time, and Elektra Records in the United States where the album peaked at #4 and has been certified Triple Platinum (three million copies sold).\nThe album takes its name from the Marx Brothers film of the same name, which the band watched one night at the studio complex when recording. The 1976 follow-up album, A Day at the Races, was also named after a Marx brothers film.\n\nAt the time the most expensive album ever recorded, Brian May has asserted in subsequent years that the band would have disbanded had it not been successful. Upon release, A Night at the Opera was a commercial and critical success, debuting at #1 in the UK charts and remaining there for nine weeks. In the US, it debuted at #4, the band's strongest showing at that time. Reviews were generally positive. Rolling Stone wrote: \"What sets them apart is their selection of unlikely effects: acoustic piano, harp, acapella vocals, no synthesizers. Coupled with good songs. Queen's obviously the strongest contender in its field.\" Melody Maker called the album a \"must-have,\" encouraging listeners to \"turn it up loud and enjoy,\" while the Winnipeg Free Press wrote: \"The group's potential is practically limitless, indicating that Queen is destined to finally take its place among the small handful of truly major acts working in rock today.\" Robert Christgau, however, was less enthusiastic, giving the album a \"B-\" rating. Later reviews in notable publications such as Allmusic, Mojo and Q Magazine, have seen the album receive five-star ratings. Allmusic wrote: \"It's prog rock with a sense of humor as well as dynamics, and Queen never bettered their approach anywhere else.\" Mojo called the album \"An imperial extravaganza, a cornucopia; a band of hungrily competitive individualists on a big roll of friendship and delight.\" Q were similarly receptive, writing: \"It's a record to which all four contributed fine songs, and one of extremes -- among the crashing rock and proggy wigouts were love songs, acoustic whimsy, a trad-jazz pastiche and a brace of vaudeville show tunes.\" Uncut awarded the album only 3/5 stars, but offered a mostly positive review, commending \"the extent of the band's barmy diversity.\"\n\nSince its release, A Night at the Opera has frequently been cited as one of Queen's finest works. In particular, Bohemian Rhapsody, the album's centerpiece, received acclaim and is still considered among the greatest songs ever recorded. The British public voted it the 13th greatest album of all time in a 2004 Channel 4 poll, and the 9th greatest number one album of all time in a 2006 BBC poll. The album has also ranked highly in international polls and has frequently appeared in \"greatest albums\" lists reflecting the opinions of critics.\n\nA Night at the Opera appears alongside other Queen albums Queen II and Sheer Heart Attack in the book 1001 Albums You Must Hear Before You Die.\n\nThe album was first time re-released in the US on Hollywood Records on 3 September 1991 with two bonus remixes, as part of a complete re-release of all Queen albums.\n\nOn 30 April 2002 the album was again re-released on DVD-Audio with a 5.1-channel mix in Dolby Digital and DTS surround sound.\n\nOn 21 November 2005 it was once more re-released to celebrate the 30th anniversary of the album and its first single, \"Bohemian Rhapsody\". This release is accompanied by a DVD-Video disc with the same track listing featuring the original videos, old and new concert footage (including \"'39\" from the Queen + Paul Rodgers tour and Brian May on the roof of Buckingham Palace playing \"God Save the Queen\") and audio commentary by all four bandmembers. It was on this commentary (and on In the Studio with Redbeard, which devoted an episode to A Night at the Opera) that May stated that had the album not been a success, Queen certainly would have disbanded. <a href=\"http://www.last.fm/music/Queen/A+Night+at+the+Opera\">Read more on Last.fm</a>. User-contributed text is available under the Creative Commons By-SA License; additional terms may apply.",
        "published": "29 Nov 2008, 18:16",
        "summary": "A Night at the Opera is a 1975 album by English rock band Queen. It was produced by Roy Thomas Baker and Queen, and reportedly was, at the time of its release, the most expensive album ever made. It was originally released by EMI in the UK where it topped the charts for nine weeks, a record at the time, and Elektra Records in the United States where the album peaked at #4 and has been certified Triple Platinum (three million copies sold).\nThe album takes its name from the Marx Brothers film of the same name <a href=\"http://www.last.fm/music/Queen/A+Night+at+the+Opera\">Read more on Last.fm</a>."
    }
}
